plans/example-browser/playwright-runner.js

```diff
@@ -0,0 +1,41 @@
+const { chromium, firefox } = require('playwright')
+
+module.exports = (module) => {
```

Comment:
Since this function forms the base of the tests, I would like to see comments on its behavior and usage; same goes for `runTestFn`

Reply:
Sure, is there a particular format of docstring you expect here? Wasn't under the impression that this needed any comments given the fact that it's (a) an example and (b) very small in scope. But sure, it also cannot hurt, especially for an example. So I'll add it with pleasure.

runFn is a private function though, but I'll never the less add a comment.

Note that for the second example (the one I still need to add and which will run testground-sdk from within the browser) the flow will still be a bit different and slightly more complex)

Reply:
The functions are clear enough, but I have future developers in mind - when someone wishes to use testground browser tests to their own project, they will use this example (or others, if we replace this). It makes sense to keep the example as clear and documented and possible, so in the future, people don't have to question why / how something was done.

Does this make sense?

Reply:
Thanks for the note @brdji, huge 👍 to aiming for more documentation and more comments,

A few notes to make sure we're staying on the pragmatic side of things:

- We expect this code to change a lot over the next few weeks, so it's fine to hold off "user-oriented" documentation (this is the `testground:edge` version). A `README.md` would be a great start with an example or two coming from the integration test, no blockers for me.
- We need "maintainers-oriented" comments when a piece of code hides some specialized knowledge or some context that cannot be guessed by a maintainer having a quick read. If we feel like a piece of code is self-explanatory, let's not worry about comments.

Reply:
Added the desired comments.